tests/plotting/test_plot_manager.py

```diff
@@ -76,10 +76,64 @@ def drop(self, path: Path):
         del self.plots[self.plots.index(self.path / path.name)]
 
 
+class PlotRefreshTester:
+    plot_manager: PlotManager
+    expected_result: PlotRefreshResult
+    expected_result_matched: bool
+
+    def __init__(self, root_path: Path):
+        self.plot_manager = PlotManager(root_path, self.refresh_callback)
+        self.expected_result = PlotRefreshResult()
+        self.expected_result_matched = False
```

Comment:
the assignments to `self.expected_result_matched` and `expected_result` are "dead". they will always be overwritten in `run()`.

It doesn't really seem like these belong as members in this class, but this class also seems to be a round-about way to include these in the closure of the callback.

Reply:
Dropped.